fastlane_bot/helpers/tradeinstruction.py

```diff
@@ -206,7 +206,7 @@ def get_custom_int(self) -> int:
         pool = self.pool
         custom_int = 0
         if self.exchange_name in self.ConfigObj.UNI_V3_FORKS:
-            custom_int = int(Decimal(pool.fee_float) * Decimal("1000000"))
+            custom_int = int(pool.fee)
```

Comment:
How is this change related to this PR?

Reply:
Nile V3 includes a fee override.

This PR stores the fee override for the pool in the `fee_float` field, and the original fee (which is necessary for Uniswap V3 details such as tick-spacing etc) in the `fee` field.

The Fast Lane arb contract must know the `fee` of the Uni V3 pool being traded on, so using the `fee_float` to calculate the `fee` is no longer valid in instances of override. 

Consequently the `fee` field is now used as it does not change.

Reply:
I did not understand this description.
But Nile V3 should not be handled differently than any other DEX, other than in the dedicated sections (for example, in the uniswap_v3 module, where all of these DEXs are handled).

Reply:
Correct, and to handle it the same, we must now use the `fee` field - which remains constant, instead of the `fee_float` which has an override for Nile V3.